rotkehlchen/api/v1/schemas.py

```diff
@@ -188,19 +190,29 @@ def make_ethereum_transaction_query(  # pylint: disable=no-self-use
     ) -> Dict[str, Any]:
         address = data.get('address')
         order_by_attribute = data['order_by_attribute'] if data['order_by_attribute'] is not None else 'timestamp'  # noqa: E501
+        protocols, asset = data['protocols'], data['asset']
+        if protocols is None or len(protocols) == 0:
+            protocols = None
```

Comment:
But with this condition this you are reassigning `protocols=None`. What I would have written is
```suggestion
        if protocols is not None and len(protocols) == 0:
            protocols = None
```

Reply:
Alright